src/components/icon/icon.jsx

```diff
@@ -11,6 +11,13 @@ export default function Icon(props) {
     props.currentColor === 'skip' ? '' : styles[`color-${props.currentColor}`],
   ].filter(Boolean);
 
+  const tabindex = props.tabable ? '0' : undefined;
+  const onKeyDown = (event) => {
+    if (event.key === 'Enter') {
```

Comment:
We want to support both: `Enter` and `Space`


https://www.w3.org/TR/wai-aria-practices-1.1/#button
https://www.w3.org/TR/wai-aria-practices-1.1/examples/button/button.html